tests/Configs/Default/CATChem_emission.yml

```diff
@@ -22,3 +22,13 @@ dust:
   DU10:
     long_name: "Dust Species Coarse Mode"
     map: ["dust1", "dust2"]
```

Comment:
What do you think about adding the MW in these yaml files instead of hardcoded in the code?